experiments/utils.py

```diff
@@ -8,8 +8,8 @@
 from pygments.formatters import HtmlFormatter
 from IPython.core.display import HTML
 
-
-def rng_generator(master_seed=1):
+MASTER_SEED=1
+def rng_generator():
```

Comment:
I changed this because it was misleading in my opinion.
Basically, the argument `master_seed` is only taken into account the first time the function is called, when generating the seed sequence, so if it's called multiple times with different seeds the subsequent seeds are ignored. 
I think it's better to have it like this.